eoc_journal/api_client.py

```diff
@@ -237,3 +248,20 @@ def get_cohort_average_progress(self):
         if data:
             return data.get('course_avg', None)
         return None
+
+    def get_user_proficiency(self):
+        """
+        Fetches and returns the user's and average course proficiency scores.
+        """
+        data = self._get_grades_leader_metrics()
+        if data is None:
+            return None
+
+        user_grade = data.get('user_grade')
+        course_avg = data.get('course_avg')
+        if user_grade is None or course_avg is None:
+            return None
+
+        user_grade = int(round(user_grade * 100.0))
+        course_avg = int(round(course_avg * 100.0))
```

Comment:
Nit: Maybe `floor` instead of rounding (can just remove the `round`), so that 100% will only ever display when everything is perfectly complete?